fixtures/cameo/hydrabeam-clhb400w-rgbw.json

```diff
@@ -0,0 +1,803 @@
+      "capability": {
+        "type": "EffectSpeed",
+        "speedStart": "slow",
+        "speedEnd": "fast"
+      }
+    },
+    "Master Red": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Master Green": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Master Blue": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "Master White": {
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    },
+    "Head speed": {
+      "capability": {
+        "type": "PanTiltSpeed",
+        "speedStart": "fast",
+        "speedEnd": "slow"
+      }
+    },
+    "Sound mode / Reset": {
+      "capabilities": [
+        {
+          "dmxRange": [0, 199],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [200, 209],
+          "type": "Maintenance",
+          "comment": "Reset"
+        },
+        {
+          "dmxRange": [210, 240],
+          "type": "NoFunction"
+        },
+        {
+          "dmxRange": [241, 255],
+          "type": "Effect",
+          "effectName": "Sound Control",
+          "soundControlled": true
+        }
+      ]
+    }
+  },
+  "modes": [
+    {
+      "name": "6 Channel",
+      "shortName": "6ch",
+      "channels": [
+        "Master Dimmer",
+        "Strobe",
+        "Show Mode",
+        "Show mode speed / Sound sensitivity",
+        "Light control",
+        "Auto color change speed"
+      ]
+    },
+    {
+      "name": "10 Channel",
+      "shortName": "10ch",
+      "channels": [
+        "Master Pan",
+        "Master Tilt",
+        "Master Dimmer",
+        "Strobe",
+        "Show Mode",
+        "Show mode speed / Sound sensitivity",
+        "Light control",
+        "Auto color change speed",
+        "Master Pan fine",
+        "Master Tilt fine"
+      ]
+    },
+    {
+      "name": "19 Channel",
+      "shortName": "19ch",
+      "channels": [
+        "Master Dimmer",
+        "Strobe",
+        "Master Red",
+        "Master Green",
+        "Master Blue",
+        "Master White",
+        {
+          "insert": "matrixChannels",
+          "repeatFor": "eachPixelABC",
+          "channelOrder": "perPixel",
+          "templateChannels": [
+            "Pan $pixelKey",
+            "Tilt $pixelKey",
+            "Dimmer $pixelKey"
+          ]
+        },
+        "Head Speed"
+      ]
+    },
+    {
+      "name": "32 Channel",
+      "shortName": "32ch",
+      "channels": [
+        "Master Dimmer",
+        "Strobe",
+        {
+          "insert": "matrixChannels",
+          "repeatFor": "eachPixelABC",
+          "channelOrder": "perPixel",
+          "templateChannels": [
+            "Pan $pixelKey",
+            "Pan $pixelKey fine",
+            "Tilt $pixelKey",
+            "Tilt $pixelKey fine",
+            "Head Speed $pixelKey",
```

Comment:
I think Head 1 is supposed to use 'Head speed' instead of `Head speed simple`, however I'm not sure how to override this in the ` "insert": "matrixChannels"` section.

Reply:
I think I've resolved this by using a second 'insert' block. The first has `repeatFor: [` and the second has `repeatFor: [2, 3, 4]` which I'm hoping works.